frontend/pages/entities/[slug].page.tsx

```diff
@@ -21,6 +21,20 @@ const EntityRelatedQuoteCard = styled(Card)({
   color: "black",
 });
 
+const EntityRelatedEntityCard = styled(Card)({
+  quotes: '"“" "”" "‘" "’"',
+  cursor: "pointer",
+  position: "relative",
+  textOverflow: "ellipsis",
+  minHeight: "1rem",
+  marginBottom: "1rem",
+  width: "17rem",
+  color: "black",
+  "&:hover": {
+    backgroundColor: "#c4e1ff",
+  },
+});
+
```

Comment:
Let's extract all the styles that are shared between this and `EntityRelatedQuoteCard` to a `sharedStyles` object.
E.g.
```ts
const sharedStyles: CSSProperties = {
  cursor: "pointer",
}

const EntityRelatedQuoteCard = styled(Card)({
  ...sharedStyles,
  otherStyles
})

const EntityRelatedEntityCard = styled(Card)({
  ...sharedStyles,
  otherStyles
})
```